### ISSUE TITLE
Add notification types for when something is taken/put/both into or out of chests.

### ISSUE THREAD

**Issue by GavJenks** [Civcraft/JukeAlert#11]
Instead of just "Accessed", have 4 different types of reports, depending:

"Opened" means opened but didn't do anything.
"Added Items" means opened and put stuff in only.
"Removed Items" means opened and took stuff out only.
"Exchanged Items" means opened and did a mixture of taking and adding stuff.

It doesn't spam the logs or take up any more disk space. Only one of those is still ever logged for each opening of the block, it's just that it's less annoyingly vague.

Opening an unlocked chest and then walking away is not what most people would consider a crime. Stealing a bunch of stuff is. Thus, the plugin would be much more useful if it distinguished. Especially when there might be 7 people who opened the chest in between checking the snitch. And then something is missing...

**Comment by rourke750** [comment]
http://imgur.com/LPPOiMK
https://github.com/Civcraft/JukeAlert/commit/cb7fe0e42c75dccaba3060433aea374edbded398

How Juke Alert is set up you can't really set custom messages.

**Comment by erocs** [comment]
It would just be additional values in the enum for opened/added/removed/exchanged. Even the item ID of what was added/removed could be placed into the material field instead of the container block type, since it's known that a container at location X is being accessed.

**Comment by rourke750** [comment]
So would it be four extra lines per exchange in the log?

**Comment by erocs** [comment]
Yes, although I missed "Only one of those is still ever logged for each opening of the block". With the current implementation there is no log consolidation so it would create log spam. Probably not worth it with the Exchange notification, but GavJenks can argue for it.

**Comment by GavJenks** [comment]
I am thinking I didn't explain clearly enough. 

It was poor word choice with "exchanged" -- I did not mean "used item exchange," and you seem to have posted the same screenshot for here and the itemexchange logging suggestion I made. Whichever one is actually implemented no, thanks! But two different concepts. One is logging itemexchanges. The other is an indication that items were both placed in and removed from a chest while it was opened by a player. Perhaps "Swapped" or whatever, I don't know. The idea is to therefore cover all logical possibilities, each with just ONE log entry, so that the spam is not any higher than it is right now:

Did not open a chest -- isn't logged
Opened a chest but did nothing with the contents -- "opened"
Opened a chest and put stuff in it but did not remove stuff -- "removed item(s)"
Opened a chest and removed stuff but didn't put anything new in -- "added item(s)"
Opened a chest and within that one interaction, added things and removed other things -- "swapped item(s)" or whatever terminology.

Every chest access event therefore generates exactly one log entry, just like now, but it tells you whether they actually stole things, which is what people actually care about. The way it stands now, i could open a chest, PUT diamonds in it, and then get pearled for being a thief...
